src/ApplePayGateway.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Omnipay\Vindicia;
+
+
+class ApplePayGateway extends AbstractVindiciaGateway
+{
+
+    /**
+     * Get the gateway parameters.
+     *
+     * @return array
+     */
+    public function getDefaultParameters()
+    {
+        return array(
+            'username' => '',
+            'password' => '',
+            'testMode' => false,
+        );
+    }
+
+    /**
+     * Get the gateway name.
+     *
+     * @return string
+     */
+    public function getName()
+    {
+        return 'Vindicia ApplePay';
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getUsername()
+    {
+        return $this->getParameter('username');
+    }
+
+    /**
+     * @return AbstractVindiciaGateway
+     */
+    public function setUsername($value)
+    {
+        return $this->setParameter('username', $value);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getPassword()
+    {
+        return $this->getParameter('password');
+    }
+
+    /**
+     * @return AbstractVindiciaGateway
+     */
+    public function setPassword($value)
```

Comment:
these are all in the base class

Reply:
Right, forgot to remove these. Was trying to see if I could add them here so that I wouldn't have to in authorize(), but it checks for them in that method so... eh